.github/workflows/go-test.yml

```diff
@@ -30,19 +33,22 @@ jobs:
         uses: ./.github/actions/go-test-setup
         if: hashFiles('./.github/actions/go-test-setup') != ''
       - name: Run tests
-        uses: protocol/multiple-go-modules@v1.2
+        uses: protocol/multiple-go-modules@v1.3
+        shell: ${{ matrix.os.shell }}
```

Comment:
https://github.com/libp2p/go-openssl/actions/runs/1489688026/workflow

```
The workflow is not valid. .github/workflows/go-test.yml (Line: 37, Col: 9): Unexpected value 'shell' .github/workflows/go-test.yml (Line: 43, Col: 9): Unexpected value 'shell'
```

Reply:
The `shell` param belongs under `with`. I'll update the source PR in .github repo later.